data/json/overmap/multitile_city_buildings.json

```diff
@@ -36,6 +36,16 @@
       { "point": [ 0, 0, -1 ], "overmap": "basement" }
     ]
   },
+  {
+    "type": "city_building",
+    "id": "house_04",
+    "locations": [ "land" ],
+    "overmaps": [
+      { "point": [ 0, 0, 0 ], "overmap": "house_04_north" },
+      { "point": [ 0, 0, 1 ], "overmap": "house_04_roof_north" },
+      { "point": [ 0, 0, -1 ], "overmap": "basement" }
```

Comment:
Should we make a custom basement for house_04 and all these houses? Not in this PR but I could probably whip something up quickly.

Reply:
none of the stairs are going to line up, you're call if you want to take them on.  Right now they all pull from the general basement list.